schematools/importer/base.py

```diff
@@ -370,8 +427,42 @@ def table_factory(
             col_kwargs["primary_key"] = True
             col_kwargs["nullable"] = False
             col_kwargs["autoincrement"] = False
-
+        
+        
         id_postfix = "_id" if field.relation else ""
-        columns.append(Column(f"{field_name}{id_postfix}", col_type, **col_kwargs))
+        columns.append(Column(f"{field_name}{id_postfix}", col_type, **col_kwargs))    
+    
+    
+    return {db_table_name: Table(db_table_name, metadata, *columns), **identifier_index, **sub_tables} 
+
+    
 
-    return {db_table_name: Table(db_table_name, metadata, *columns), **sub_tables}
+def define_identifier_index(db_table_name: str, dataset_table: DatasetTableSchema) -> Dict: #adjusted   
+    """Generate one or more SQLAlchemy Index objects to work with the JSON Schema
+
+    :param db_table_name: The full table name: [name_dataset]_ [name_table]
+    :param dataset_table: The Amsterdam Schema definition of the table
+    
+    In the JSON Schema definition of the table, an identifier arry maybe definied.
+    I.e. "identifier": ["identificatie", "volgnummer"]
+    
+    The purpose is to set, besides the (single) PK, the (compound) Alternate Key to uniquely identify a table record.
```

Comment:
De key is wel een alternate key, maar toch een compound (composite) key? In dat laatste geval vormen meerdere kolommen samen de PK, maar dat is hier niet zo.

Reply:
Ja is niet goed verwoord. Ik heb het aangepast door deze zin er zelfs uit te halen. En qua uitleg nu gericht op het aanmaken van een index op de opgeven identifier